### ISSUE TITLE
运行情感分析训练样例官方给的训练命令，在CPU服务器运行，基本上处于卡住不动状态

### ISSUE THREAD

**Issue by zhangshenghang** [PaddlePaddle/PaddleNLP#6929]
### 请提出你的问题

运行官方给的训练命令，在CPU服务器运行，基本上处于卡住不动状态，是什么原因？

官方给的数据样例数据，数据很小，运行了12个小时还没运行完成。CPU服务器不支持吗(官方案例给的是GPU运行)？

运行的命令
`
python -u -m paddle.distributed.launch  finetune.py \
--train_path ./data/train.json \
--dev_path ./data/dev.json \
--save_dir ./checkpoint \
--learning_rate 1e-5 \
--batch_size 1 \
--max_seq_len 10 \
--num_epochs 1 \
--model uie-senta-base \
--seed 1000 \
--logging_steps 10 \
--valid_steps 100 \
--device cpu
`

运行结果
`/root/anaconda3/envs/nlp4/lib/python3.8/site-packages/_distutils_hack/__init__.py:33: UserWarning: Setuptools is replacing distutils.
  warnings.warn("Setuptools is replacing distutils.")
[2023-09-05 17:53:14,890] [    INFO] - We are using (<class 'paddlenlp.transformers.ernie.tokenizer.ErnieTokenizer'>, False) to load 'uie-senta-base'.
[2023-09-05 17:53:14,891] [    INFO] - Already cached /root/.paddlenlp/models/uie-senta-base/ernie_3.0_base_zh_vocab.txt
[2023-09-05 17:53:14,931] [    INFO] - tokenizer config file saved in /root/.paddlenlp/models/uie-senta-base/tokenizer_config.json
[2023-09-05 17:53:14,932] [    INFO] - Special tokens file saved in /root/.paddlenlp/models/uie-senta-base/special_tokens_map.json
[2023-09-05 17:53:14,933] [    INFO] - Already cached /root/.paddlenlp/models/uie-senta-base/model_state.pdparams
[2023-09-05 17:53:14,933] [    INFO] - Loading weights file model_state.pdparams from cache at /root/.paddlenlp/models/uie-senta-base/model_state.pdparams
[2023-09-05 17:53:17,397] [    INFO] - Loaded weights file from disk, setting weights to model.
[2023-09-05 17:53:28,211] [    INFO] - All model checkpoint weights were used when initializing UIE.

[2023-09-05 17:53:28,212] [    INFO] - All the weights of UIE were initialized from the model checkpoint at uie-senta-base.
If your task is similar to the task the model of the checkpoint was trained on, you can already use UIE for predictions without further training.
/root/anaconda3/envs/nlp4/lib/python3.8/site-packages/paddlenlp/transformers/tokenizer_utils_base.py:2478: FutureWarning: The `max_seq_len` argument is deprecated and will be removed in a future version, please use `max_length` instead.
  warnings.warn(
/root/anaconda3/envs/nlp4/lib/python3.8/site-packages/paddlenlp/transformers/tokenizer_utils_base.py:1878: FutureWarning: The `pad_to_max_length` argument is deprecated and will be removed in a future version, use `padding=True` or `padding='longest'` to pad to the longest sequence in the batch, or use `padding='max_length'` to pad to a max length. In this case, you can give a specific length with `max_length` (e.g. `max_length=45`) or leave max_length to None to pad to the maximal input size of the model (e.g. 512 for Bert).
  warnings.warn(
[2023-09-05 20:37:40,974] [    INFO] - global step 10, epoch: 1, loss: 0.00179, speed: 0.00 step/s
[2023-09-05 23:26:42,289] [    INFO] - global step 20, epoch: 1, loss: 0.00197, speed: 0.00 step/s
[2023-09-06 02:08:14,692] [    INFO] - global step 30, epoch: 1, loss: 0.00228, speed: 0.00 step/s
[2023-09-06 04:51:48,249] [    INFO] - global step 40, epoch: 1, loss: 0.00196, speed: 0.00 step/s
[2023-09-06 07:31:34,211] [    INFO] - global step 50, epoch: 1, loss: 0.00190, speed: 0.00 step/s`

![image](https://github.com/PaddlePaddle/PaddleNLP/assets/29418975/13e636bd-f57c-4737-aea2-e6ba85737066)

**Comment by XJAlive** [comment]
![image](https://github.com/PaddlePaddle/PaddleNLP/assets/35095965/bd874616-80f2-46f4-8636-0d8b16aeba43)
我是提示Pod completed了但是没有输出，用官方的酒店属性标注demo是正常的，我自己用label_studio导出的数据训练没有模型生成，蹲一个后续

**Comment by XJAlive** [comment]
> ### 请提出你的问题
> 运行官方给的训练命令，在CPU服务器运行，基本上处于卡住不动状态，是什么原因？
> 
> 官方给的数据样例数据，数据很小，运行了12个小时还没运行完成。CPU服务器不支持吗(官方案例给的是GPU运行)？
> 
> 运行的命令 `python -u -m paddle.distributed.launch finetune.py \ --train_path ./data/train.json \ --dev_path ./data/dev.json \ --save_dir ./checkpoint \ --learning_rate 1e-5 \ --batch_size 1 \ --max_seq_len 10 \ --num_epochs 1 \ --model uie-senta-base \ --seed 1000 \ --logging_steps 10 \ --valid_steps 100 \ --device cpu`
> 
> 运行结果 `/root/anaconda3/envs/nlp4/lib/python3.8/site-packages/_distutils_hack/**init**.py:33: UserWarning: Setuptools is replacing distutils. warnings.warn("Setuptools is replacing distutils.") [2023-09-05 17:53:14,890] [ INFO] - We are using (<class 'paddlenlp.transformers.ernie.tokenizer.ErnieTokenizer'>, False) to load 'uie-senta-base'. [2023-09-05 17:53:14,891] [ INFO] - Already cached /root/.paddlenlp/models/uie-senta-base/ernie_3.0_base_zh_vocab.txt [2023-09-05 17:53:14,931] [ INFO] - tokenizer config file saved in /root/.paddlenlp/models/uie-senta-base/tokenizer_config.json [2023-09-05 17:53:14,932] [ INFO] - Special tokens file saved in /root/.paddlenlp/models/uie-senta-base/special_tokens_map.json [2023-09-05 17:53:14,933] [ INFO] - Already cached /root/.paddlenlp/models/uie-senta-base/model_state.pdparams [2023-09-05 17:53:14,933] [ INFO] - Loading weights file model_state.pdparams from cache at /root/.paddlenlp/models/uie-senta-base/model_state.pdparams [2023-09-05 17:53:17,397] [ INFO] - Loaded weights file from disk, setting weights to model. [2023-09-05 17:53:28,211] [ INFO] - All model checkpoint weights were used when initializing UIE.
> 
> [2023-09-05 17:53:28,212] [ INFO] - All the weights of UIE were initialized from the model checkpoint at uie-senta-base. If your task is similar to the task the model of the checkpoint was trained on, you can already use UIE for predictions without further training. /root/anaconda3/envs/nlp4/lib/python3.8/site-packages/paddlenlp/transformers/tokenizer_utils_base.py:2478: FutureWarning: The `max_seq_len` argument is deprecated and will be removed in a future version, please use `max_length` instead. warnings.warn( /root/anaconda3/envs/nlp4/lib/python3.8/site-packages/paddlenlp/transformers/tokenizer_utils_base.py:1878: FutureWarning: The `pad_to_max_length` argument is deprecated and will be removed in a future version, use `padding=True` or `padding='longest'` to pad to the longest sequence in the batch, or use `padding='max_length'` to pad to a max length. In this case, you can give a specific length with `max_length` (e.g. `max_length=45`) or leave max_length to None to pad to the maximal input size of the model (e.g. 512 for Bert). warnings.warn( [2023-09-05 20:37:40,974] [ INFO] - global step 10, epoch: 1, loss: 0.00179, speed: 0.00 step/s [2023-09-05 23:26:42,289] [ INFO] - global step 20, epoch: 1, loss: 0.00197, speed: 0.00 step/s [2023-09-06 02:08:14,692] [ INFO] - global step 30, epoch: 1, loss: 0.00228, speed: 0.00 step/s [2023-09-06 04:51:48,249] [ INFO] - global step 40, epoch: 1, loss: 0.00196, speed: 0.00 step/s [2023-09-06 07:31:34,211] [ INFO] - global step 50, epoch: 1, loss: 0.00190, speed: 0.00 step/s`
> 
> ![image](https://user-images.githubusercontent.com/29418975/265869842-13e636bd-f57c-4737-aea2-e6ba85737066.png)

兄弟加个联系方式，讨论下？wx:18802092131

**Comment by ChiayenGu** [comment]
> ![image](https://user-images.githubusercontent.com/35095965/265874573-bd874616-80f2-46f4-8636-0d8b16aeba43.png) 我是提示Pod completed了但是没有输出，用官方的酒店属性标注demo是正常的，我自己用label_studio导出的数据训练没有模型生成，蹲一个后续

我也是，蹲蹲后续
